src/rocksdb_options.rs

```diff
@@ -740,6 +740,12 @@ impl DBOptions {
         }
     }
 
+    pub fn set_fifo_compaction_options(&mut self, fifo_opts: FifoCompactionOptions) {
```

Comment:
I think this function should belong to `ColumnFamilyOptions`.

Reply:
I agree. I think it was an oversight by rocksdb team. I think we shouldn't deviate from the original api though.

Reply:
In this repo, we distinguish `DBOptions` and `ColumnFamilyOptions` although they both wrap RocksDB's `Options`.

Reply:
Good point. I changed it now.